presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -164,6 +164,22 @@ public void testSTIsEmpty()
         assertFunction("ST_IsEmpty(ST_GeometryFromText('POLYGON EMPTY'))", BOOLEAN, true);
     }
 
+    @Test
+    public void testSTIsSimple()
+    {
+        assertFunction("ST_IsSimple(ST_GeometryFromText('POLYGON EMPTY'))", BOOLEAN, true);
+        assertFunction("ST_IsSimple(ST_GeometryFromText('POINT (1.5 2.5)'))", BOOLEAN, true);
+        assertFunction("ST_IsSimple(ST_GeometryFromText('MULTIPOINT (1 2, 2 4, 3 6, 4 8)'))", BOOLEAN, true);
```

Comment:
Could you add a test for a multi-point that's not simple, e.g. multi-point with duplicate points.